cli/azd/pkg/infra/provisioning/manager.go

```diff
@@ -255,19 +255,25 @@ func (m *Manager) ensureLocation(ctx context.Context, deployment *Deployment) (s
 	return location, nil
 }
 
-func (m *Manager) runAction(title string, interactive bool, action func(spinner *spin.Spinner) error) error {
+func (m *Manager) runAction(ctx context.Context, title string, interactive bool, action func(ctx context.Context, spinner *spin.Spinner) error) error {
 	var spinner *spin.Spinner
 
 	if interactive {
-		spinner = spin.NewSpinner(title)
+		// Check whether there is already a spinner active in the current context
+		spinner = spin.GetSpinner(ctx)
```

Comment:
Should we be reasserting the title in this case?

Reply:
Added `GetOrCreateSpinner` that takes in the desired title and sets it.